dbbackup/management/commands/dbbackup.py

```diff
@@ -43,16 +45,29 @@ def handle(self, **options):
         self.compress = options.get('compress')
         self.encrypt = options.get('encrypt')
         self.storage = BaseStorage.storage_factory()
-        database_keys = (self.database,) if self.database else dbbackup_settings.DATABASES
-        for database_key in database_keys:
-            database = settings.DATABASES[database_key]
-            self.dbcommands = DBCommands(database)
+        if options.get('mongo'):
+            # Mongo specific handling.
+            # TODO: Add support for multiple mongo db defined in settings.
```

Comment:
Is it that it's too much work to setup a connection and access objects with django-mongodb-engine ?

Because if you could implement backup support following this conventional django db backend + settings.DATABASES, it would be the same seamless interface that the other backends are running.

One of the points of this project is to follow the database agnostic pattern... so you can just like run the command and not worry about what type of database it's connecting to :)